docs/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview.md

```diff
@@ -4,10 +4,10 @@ title: Overview
 description: Take a closer look at the connectors available in Camunda Platform 8.
 ---
 
-This section gives an overview of the **connectors** available in Camunda Platform 8:
+This section gives an overview of the **out-of-the-box connectors** available in Camunda Platform 8:
 
 - [REST connector](rest.md) - Make a request to a REST API and use the response in the next steps of your process.
 - [SendGrid connector](sendgrid.md) - Quickly send emails from your BPMN processes.
 - [Slack connector](slack.md) - Send messages to channels or users in your [Slack](https://slack.com) workspace from your BPMN process.
 
-In addition to this section on connectors, we recommend reviewing [connector Secrets](../../../../console/manage-clusters/manage-secrets.md).
+In addition to this section on connectors, we recommend reviewing [connector Secrets](../../../console/manage-clusters/manage-secrets.md).
```

Comment:
@MaxTru I am really confused by the naming. I thought connectors where a product name, thus starting with a capital letter, while secrets is something general, thus lowercase. But from this docs, it seems like it is the other way around.

I am asking since I am writing the docs for the Connector Template Editor and want to be consistent. What is the reasoning for this style here?

Reply:
I believe this is just a typo, oversight. We can update and capitalise Connectors, and lower case Secrets. Thanks.

Reply:
@christinaausley, maybe you can help here? Or maybe this is a misunderstanding between what is a product name and what is not.

Reply:
Agreed here that secrets should be lowercase.

As far as "connectors" goes -- this is purely up to preference. All of our products (Camunda Platform 8, Camunda Platform 7, Self-Managed) are capitalized, as are our key components (Optimize, Operate, etc.) Think of this like "Microsoft Excel" or "Microsoft Word" -- they are all key applications of a company, hence capitalization.

If we create concepts that are *supplemental* to our key components (think Optimize's "reports" and "dashboards" or "clusters", these are lowercase.

Thus, why I circle back to preference. Do we consider connectors supplemental to Modeler, or their own, independent product?

Reply:
+1 for `secrets` and `Connectors`

Reply:
Ok I see that we do not capitalize `Connectors` on our websites. We only capitalize  `Integration Framework`. I will stick to that https://camunda.com/platform/modeler/connectors/

Reply:
Great! I'll ensure this is mirrored in the [PR I've launched](https://github.com/camunda/camunda-platform-docs/pull/1174). @PreetiNirwal -- feel free to take a look at the files changed for a closer look at all of my changes.

Also note I've launched a [PR to lowercase secrets](https://github.com/camunda/camunda-platform-docs/pull/1175) and will launch a subsequent PR to ensure all instances of Connectors are uppercase.

Reply:
Pulling all the threads together. Let's follow up on this Slack thread with next steps - https://camunda.slack.com/archives/C026U8GBNSW/p1660769206379089